### PR TITLE
feat(interview-prep): Porting Rosetta problems

### PR DESCRIPTION
**Pull request by NitronR**
#### Pre-Submission Checklist

- [ ] Your pull request targets the `dev` branch.
- [ ] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/challenge-tests`)
- [ ] All new and existing tests pass the command `npm test`.
- [ ] Use `npm run commit` to generate a conventional commit message.
    Learn more here: <https://conventionalcommits.org/#why-use-conventional-commits>
- [ ] The changes were done locally on your machine and NOT GitHub web interface.
    If they were done on the web interface you have ensured that you are creating conventional commit messages.

#### Checklist:

- [ ] Tested changes locally.
- [ ] Addressed currently open issue (replace XXXXX with an issue no in next line)

Closes #XXXXX

### CODE REVIEW

**Comment by scissorsneedfoodtoo** [comment]
@NitronR, sorry for the delay in reviewing these. Will leave some suggestions as comments if I come across anything.

File: challenges/08-coding-interview-prep/rosetta-code.json [hunk 1]
```diff
@@ -13377,6 +13377,1144 @@
...
+          "ext": "js",
+          "name": "index",
+          "contents": [
+            "function solveApollonius (c1, c2, c3, s1, s2, s3) {",
+            "  // Good luck!",
+            "}"
+          ],
+          "head": [],
+          "tail": [
+            "var tests=[",
```
Comment (scissorsneedfoodtoo) [inline challenges/08-coding-interview-prep/rosetta-code.json]: Would it be possible to change this to a unique name? Maybe solveApolloniusTests. Turns out that having an array with the name tests in the tail causes all the tests to fail, as it calls the tests object instead.

**Comment by scissorsneedfoodtoo** [comment]
Other than those arrays, LGTM! Just a few minor things here and there like lists that can be fixed up later.

**Comment by NitronR** [comment]
@scissorsneedfoodtoo Changes done.

**Comment by scissorsneedfoodtoo** [comment]
@NitronR, alright, thank you for making those changes so quickly! Everything LGTM.

There are still some earlier problems with arrays that need to be renamed, but that's something we can do later.

**Pull request merged**